app/src/main/java/com/example/una/fragments/ImpactFragment.java

```diff
@@ -54,5 +56,29 @@ public void onComplete(@NonNull Task<Void> task) {
             }
 
         });
+        LinearLayoutManager layoutManager = new LinearLayoutManager(getContext());
+        rvChallenges.setLayoutManager(layoutManager);
+        bindDataToAdapter();
+    }
+
+    // place challenges for user here
+    private ArrayList<Object> getSampleArrayList() {
+        ArrayList<Object> items = new ArrayList<>();
+        items.add("image");
+        items.add(new Challenge("Challenge 1", "Valyria"));
```

Comment:
let's have better dummy data so people can visualize what these rows would represent

Reply:
So instead of Valyria for example, put some longer text to make resemble more of a challenge description?